src/Diagnostics/FxCop/CSharp/Design/CodeFixes/CA1052CSharpCodeFixProvider.cs

```diff
@@ -39,16 +36,37 @@ public sealed override FixAllProvider GetFixAllProvider()
             if (classDeclaration != null)
```

Comment:
This code is unchanged from how I found it, and is consistent with similar code in CodeFixProviderBase and MultipleCodeFixProviderBase. We should consider deriving this fixer from CodeFixProviderBase and addressing the issue (if it is one) there.